lib/smart_answer/calculators/benefit_cap_calculator_data_query.rb

```diff
@@ -1,8 +1,28 @@
 module SmartAnswer::Calculators
   class BenefitCapCalculatorDataQuery
-    attr_reader :data
+    attr_reader :data, :benefits, :questions
     def data
       @data ||= YAML.load_file(Rails.root.join('lib', 'data', 'benefit_cap_data.yml'))
     end
+
+    def benefits
+      @benefits ||=  @data.fetch("benefits").with_indifferent_access
+    end
+
+    def questions
+      @questions ||=
+        benefits.inject(HashWithIndifferentAccess.new) do |benefits_and_questions, (key, value)|
+          benefits_and_questions[key] = value.fetch("question")
+          benefits_and_questions
+        end
+    end
+
+    def descriptions
```

Comment:
It looks as though this method isn't used until commit bf2f88fbdf995fcfc261de62f73ff9ef179a3634 ("Populate checkbox list of non-exempt benefits from YAML"). Ideally it'd be combined with that commit.